### PR TITLE
Add missing "Standard" docker.io auth retrieval [BT-226]

### PR DESCRIPTION
**Pull request by mcovarr**
See BT-226

### CODE REVIEW

**Review by kshakir — APPROVED** [review]
`- should call cache the second run of hello_private_repo (4 minutes, 14 seconds)`

via: https://travis-ci.com/github/broadinstitute/cromwell/jobs/501263464

💯👍🎉

File: centaur/src/main/resources/standardTestCases/hello_private_repo.test [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: hello_private_repo
```
Comment (breilly2) [inline centaur/src/main/resources/standardTestCases/hello_private_repo.test:1]: It's not clear to me, so just to as an obvious question: Does this new test fail without the change to `StandardLifecycleActorFactory`?

**Review by breilly2 — APPROVED** [review]
👍

File: centaur/src/main/resources/standardTestCases/hello_private_repo.test (hunk 1 above)
Comment (mcovarr) [reply centaur/src/main/resources/standardTestCases/hello_private_repo.test:1]: Yes the test will fail without these changes. The workflow would succeed without these changes, but without the added plumbing through of Docker credentials the Cromwell engine will be unable to obtain the hash for the Docker image. Consequently the call caching expected by the`testFormat: runtwiceexpectingcallcaching` will not happen and the test will fail.

**Pull request merged**